rotterdam-nlds-parent-wicket/rotterdam-nlds-wicket/src/main/java/nl/rotterdam/nl_design_system/wicket/components/dialog/RdDialogBorder.java

```diff
@@ -61,13 +66,18 @@ public abstract class RdDialogBorder extends Border {
      * </p>
      *
      * @param id the wicket ID.
-     * @param titleModel the model with the title to shown in the dialog header.
+     * @param titleModel the model with the title to show in the dialog header.
+     * @param closeIconBehaviorSupplier the supplier of the behavior that will configure the icon tag.
      * @param headingLevel the level of the heading in the dialog header. This affects the title heading element and the
      *                     icon button.
      */
-    public RdDialogBorder(@NonNull String id, @NonNull IModel<?> titleModel, @NonNull RdDialogHeadingLevel headingLevel) {
+    public RdDialogBorder(@NonNull String id,
+                          @NonNull IModel<?> titleModel,
+                          @NonNull Supplier<Behavior> closeIconBehaviorSupplier,
```

Comment:
Afdwingen dat ie serializable moet zijn, zodat gebruikers dit niet vergeten en serializatieproblemen krijgen
```suggestion
                          @NonNull SerializableSupplier<Behavior> closeIconBehaviorSupplier,
```